src/eduid_webapp/signup/views.py

```diff
@@ -51,12 +51,12 @@
 def get_config():
     tou_url = current_app.config.get('TOU_URL')
     try:
-        r = requests.get(tou_url)
+        r = requests.get(tou_url, verify=False)
         current_app.logger.debug('Response: {!r} with headers: {!r}'.format(r, r.headers))
         if r.status_code == 302:
             headers = {'Cookie': r.headers.get('Set-Cookie')}
             current_app.logger.debug('Headers: {!r}'.format(headers))
-            r = requests.get(tou_url, headers=headers)
+            r = requests.get(tou_url, headers=headers, verify=False)
```

Comment:
Why don't we want to verify the certificate? Seems wrong.

Reply:
Ditto